### PR TITLE
PERA-1335 :: fix koin view model not working for ios test app

### LINKED ISSUE #67: Bug - Koin ViewModel not working on iOS Test App
Currently the `composeTestApp` module loads up fine in Android Emulator, but `iosTestApp` does not in iPhone Emulator after this PR with `AccountsViewModel` (https://github.com/perawallet/pera-android/pull/53).  Below is the error you should see if you try to run the iOS app in Android Studio

Can you investigate how we should be using Koin and viewmodels in Compose Multiplatform so the current test app will work in iOS?  If you jump back a PR or two in `dev` branch, the iOS Test App should load fine.

```
Uncaught Kotlin exception: org.koin.core.error.InstanceCreationException: Could not create instance for '[Factory: 'co.algorand.app.ui.screens.home.AccountsViewModel']'
    at 0   Pera Wallet (Compose).debug.dylib   0x104a59b3f        kfun:kotlin.Exception#<init>(kotlin.String?;kotlin.Throwable?){} + 143 
    at 1   Pera Wallet (Compose).debug.dylib   0x10547e543        kfun:org.koin.core.error.InstanceCreationException#<init>(kotlin.String;kotlin.Exception){} + 123 
    at 2   Pera Wallet (Compose).debug.dylib   0x10547f857        kfun:org.koin.core.instance.InstanceFactory#create(org.koin.core.instance.ResolutionContext){}1:0 + 1635 
    at 3   Pera Wallet (Compose).debug.dylib   0x10547f0bb        kfun:org.koin.core.instance.FactoryInstanceFactory#get(org.koin.core.instance.ResolutionContext){}1:0 + 103 
```

### PR DESCRIPTION
**Pull request by alexs60**
# Pull Request Template

## Description
- Created a dependency for SQLite Bundled
- Instructed Room Database builder to use the SQLDriver

## Related Issues
- Closes https://github.com/perawallet/pera-android/issues/67
- Closes https://algorandfoundation.atlassian.net/browse/PERA-1335

## Checklist
- [X] Have you tested your changes locally?
- [ ] Have you reviewed the code for any potential issues?
- [ ] Have you documented any necessary changes in the project's documentation?
- [ ] Have you added any necessary tests for your changes?
- [ ] Have you updated any relevant dependencies?

394953829-37481b9a-db29-496e-8e26-6c640d74734e

### CODE REVIEW

File: wallet-sdk/src/androidMain/kotlin/com/algorand/common/account/local/data/database/AccountDatabase.android.kt [hunk 1]
```diff
@@ -15,10 +15,13 @@ package com.algorand.common.account.local.data.database
 import android.content.Context
 import androidx.room.Room
 import androidx.room.RoomDatabase
+import androidx.sqlite.driver.bundled.BundledSQLiteDriver
 import kotlinx.coroutines.Dispatchers
 
 internal fun getAccountDatabase(ctx: Context): AccountDatabase {
-    return getDatabaseBuilder(ctx).build()
+    return getDatabaseBuilder(ctx)
+        .setDriver(BundledSQLiteDriver())
```
Comment (mitsinsar) [inline wallet-sdk/src/androidMain/kotlin/com/algorand/common/account/local/data/database/AccountDatabase.android.kt]: Could you please move this call into `createAccountDatabase`

File: wallet-sdk/src/iosMain/kotlin/com/algorand/common/account/local/data/database/AccountDatabase.ios.kt [hunk 2]
```diff
@@ -14,13 +14,16 @@ package com.algorand.common.account.local.data.database
...
+import androidx.sqlite.driver.bundled.BundledSQLiteDriver
 import kotlinx.cinterop.ExperimentalForeignApi
 import platform.Foundation.NSDocumentDirectory
 import platform.Foundation.NSFileManager
 import platform.Foundation.NSUserDomainMask
 
 internal fun getAccountDatabase(): AccountDatabase {
-    return createAccountDatabase().build()
+    return createAccountDatabase()
+        .setDriver(BundledSQLiteDriver())
```
Comment (mitsinsar) [inline wallet-sdk/src/iosMain/kotlin/com/algorand/common/account/local/data/database/AccountDatabase.ios.kt]: Could you please move this call into `createAccountDatabase`

**Review by mitsinsar — CHANGES_REQUESTED** [review]

**Comment by alexs60** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by alexs60** [comment]
recheck

**Comment by michaeltchuang** [comment]
recheck

**Review by michaeltchuang — APPROVED** [review]

**Review by mitsinsar — APPROVED** [review]

**Pull request merged**